css/styles/style.css

```diff
@@ -0,0 +1,155 @@
+@font-face {
+	font-family: 'codropsicons';
+	src:url('../fonts/codropsicons/codropsicons.eot');
+	src:url('../fonts/codropsicons/codropsicons.eot?#iefix') format('embedded-opentype'),
+		url('../fonts/codropsicons/codropsicons.woff') format('woff'),
+		url('../fonts/codropsicons/codropsicons.ttf') format('truetype'),
+		url('../fonts/codropsicons/codropsicons.svg#codropsicons') format('svg');
+	font-weight: normal;
+	font-style: normal;
+}
+
+.footer-container{
+    background-color: #bdced6;
+    color: #0b2239;
+    padding:0;
+    margin:0;
+    position: fixed;
+    left: 0;
+    bottom: 0;
+    width: 100%;
+    font-family: 'Lato', Arial, sans-serif;
+}
+
+#footer-main-section{
+    display: flex;
+}
+
+#footer-section1{
+    flex: 0.5;
+}
+
+#footer-section2{
+    flex: 3;
+    display: flex;
+    justify-content:space-around
+}
+
+#footer-section3{
+    display: flex;
+    justify-content: center;
+}
+
+#footer-links{
+    margin-top: 10px;
+    margin-left: 45px;
+}
+
+#footer-links ul{
+    list-style: none;
+    text-transform: uppercase;
+    border-left: 2px solid #0b2239;
+    padding-left: 10px;
+    letter-spacing: 0.2rem;
+    margin-left: 20px;
+}
+
+#footer-links ul li{
+    padding-bottom: 5px;
+    font-weight: 700;
+}
+
+#footer-links ul li:last-child {
+    padding-bottom: 0;
+}
+
+#footer-links ul li:hover {
+    color: #efeff0;
+    cursor: pointer;
+}
+
+.footer-box{
+    margin-top: 20px;
+    width: 350px;
+    height: 70px;
+    padding: 0.1rem 3rem;
+    /* outline:5px solid rgb(135, 19, 43); */
+    text-align: center;
+}
+
+.footer-box img{
+    max-width: 100%;
+    height: auto;
+    margin: 0 auto;
+    box-shadow: 0 0 12px 2px #0b2239;
+}
+
+.footer-box img:hover{
+    transform: scale(1.1);
+    display: inline-block;
+}
+
+#img-cordrop {
+    width: 100%;
+    height: auto;
+    margin: 0 auto;
+    box-shadow: 0 0 12px 2px #0b2239;
+}
+
+.footer-box a{
+    color: #0b2239;
+    font-size: 15px;
+    list-style: none;
+    text-transform: uppercase;  
+    font-weight: 700;
+    \font-weight: 700;  
```

Comment:
hey @VoronaAV23 this line seems  extra blocking our code  can we delete it please

Reply:
will fix it now